### PR TITLE
Feature/accessibility settings

### PR DESCRIPTION
**Pull request by Dremars**
New accessibility settings button and view. Accessibility settings can now be changed and saved to redux and LocalStorage. Accessibility functionality still needs to be checked with NVDA. Settings file possibly needs a breakdown to smaller components

### CODE REVIEW

**Comment by Dremars** [comment]
[Esteettömyysasetukset](https://trello.com/c/rBnGy02q/49-esteett%C3%B6myysasetukset)

**Comment by mikkoisa** [comment]
The settings button should have a visual indication that it is selected.

Näyttökuva 2019-6-17 kello 10 58 51

File: src/components/Settings/Settings.js [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+    const { classes } = this.props;
+
+    const scrollContainer = document.getElementById('SettingsContainer');
+    let container = document.getElementsByClassName('SettingsAlert')[0];
+    const content = document.getElementsByClassName('SettingsContent')[0];
+    const isAlert = !!container;
+
+    if (!isAlert) {
+      // eslint-disable-next-line prefer-destructuring
+      container = document.getElementsByClassName('SettingsConfirmation')[0];
```
Comment (mikkoisa) [inline src/components/Settings/Settings.js]: I think this line can be replaced with:
`[container] = document.getElementsByClassName('SettingsConfirmation');`
Should satisfy eslint
https://eslint.org/docs/rules/prefer-destructuring

Comment (Dremars) [reply src/components/Settings/Settings.js]: This code was rendered unnecessary when changing to use sticky elements in new [commit](https://github.com/City-of-Helsinki/servicemap-ui/pull/95/commits/df0efb4aad9ea6627a390c4ce4e0b81d697b24a3)

**Pull request merged**